raiden/ui/cli.py

```diff
@@ -204,6 +204,13 @@ def options(func):
                 type=str,
                 show_default=True,
             ),
+            option(
+                '--pathfinding-service-address',
+                help='xxx',
+                type=str,
+                default='0.0.0.0',
```

Comment:
Shouldn't it also have a port? Why 0.0.0.0? Shouldn't it be 127.0.0.1 if the user runs a local PFS?

Reply:
done we use 6000 as port